src/main/java/org/owasp/esapi/codecs/CSSCodec.java

```diff
@@ -63,13 +80,15 @@ public Character decodeCharacter(PushbackSequence<Character> input)
 	{
 		input.mark();
 		Character first = input.next();
+		System.out.println("First: " + first);
 		if (first == null || first != '\\')
 		{
 			input.reset();
 			return null;
 		}
 
 		Character second = input.next();
+		System.out.println("Second: " );
```

Comment:
This slipped in on me.  Getting it cleaned up..

Reply:
https://github.com/ESAPI/esapi-java-legacy/pull/519/commits/377d377dcd68ccefa0841ac12e980729945363d9  Cleans up the debug content